pkg/trace/config/testdata/full.ini

```diff
@@ -49,7 +49,7 @@ timeout: 4
 [trace.watchdog]
 max_memory: 1234.5
 max_cpu_percent: 85
-max_connections: 40
+max_connections: 40 # deprecated
```

Comment:
Any reason we can't just remove these lines instead of marking as deprecated?

Reply:
My reasoning was to ensure that they continue to be accepted and ignored, as sort of a backwards-compatibility safety. They could, of course, be removed.